yarn.lock

```diff
@@ -1639,7 +1639,8 @@ binary-extensions@^2.0.0:
 
 bindings@^1.5.0:
   version "1.5.0"
-  resolved "https://registry.npmjs.org/bindings/-/bindings-1.5.0.tgz"
+  resolved "https://registry.yarnpkg.com/bindings/-/bindings-1.5.0.tgz#10353c9e945334bc0511a6d90b38fbc7c9c504df"
```

Comment:
These changes are from running `yarn upgrade hot-shot` to fix an issue with `unix-dgram` sub-dependency.

![image](https://github.com/iron-fish/ironfish-api/assets/97762857/0e7d66ee-3e1f-45b5-b8bd-ea55e4687623)